meta-lmp-bsp/recipes-sota/fioefi/fioefi_%.bbappend

```diff
@@ -2,7 +2,7 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
 SRC_URI:append = " file://fioefi-soc.sh.in"
 
-RDEPENDS:${PN}:append:tegra = " setup-nv-boot-control tegra-uefi-capsules"
+RDEPENDS:${PN}:append:tegra = " setup-nv-boot-control"
```

Comment:
Why is `tegra-uefi-capsules` being removed here?

Reply:
I don't remember all details, but IIRC it was because of circular dependency.

We just need `lmp-boot-firmware` (which is the recipe that is an "envelope" for needed boot artifacts) to depend on `tegra-uefi-capsules`

Reply:
so maybe we can squash and never really include it in the creation of the bbappend